STM32Cube/Core/Src/main.c

```diff
@@ -25,6 +25,7 @@
 /* USER CODE BEGIN Includes */
 #include "printf.h"
 #include "canlib.h"
+#include "millis.h"
```

Comment:
Either include can_handler.h or extern busQueue (probably better to include right?)

Reply:
Also include message_types.h